src/RecordTable.h

```diff
@@ -108,78 +110,33 @@ class RecordTable {
     RecordTable() = default;
     virtual ~RecordTable() = default;
 
-    /**
-     * A function packing a tuple of the given arity into a reference.
-     */
+    /* @brief convert tuple to record reference for the interpreter / readstream */
```

Comment:
```suggestion
    /** Convert tuple to record reference. */
```
An asterisk was missing. Also, @brief is assumed so adding it is superfluous. I also think specifying which classes call this is not needed and will not age well.